### PR TITLE
get contestable issues through lighthouse

### PR DESCRIPTION
**Pull request by edmangimelli**
## Description of change
Resolves https://app.zenhub.com/workspaces/developervagov-5ac272fff41bfa50c0e46c48/issues/department-of-veterans-affairs/vets-contrib/3971

Picking up where https://github.com/department-of-veterans-affairs/vets-api/pull/3731 leaves off, this PR continues building the lighthouse layer of the Appeals API (instead of Kong sending requests directly to Caseflow, Kong will now (soon) forward requests to vets-api controllers, which will then call Caseflow).
This PR, specifically, adds appeals service method `get_contestable_issues` and a contestable issues controller.

## Acceptance Criteria
- [x] Tests
- [x] Appropriate logging -- using `with_monitoring` in service class
- [ ] ~Swagger docs have been updated, if applicable~ n/a
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs) --using dummy Caseflow UAT ssn (pulled from existing VCR cassette)

### CODE REVIEW

File: modules/appeals_api/app/controllers/appeals_api/application_controller.rb [hunk 1]
```diff
@@ -3,5 +3,10 @@
 module AppealsApi
   class ApplicationController < ::ApplicationController
     skip_before_action :set_tags_and_extra_context, raise: false
+
+    def render_response(resp)
+      response.headers = resp.headers
```
Comment (erluti) [inline modules/appeals_api/app/controllers/appeals_api/application_controller.rb]: we should probably remove the lighthouse api key from the headers we pass along

Comment (edmangimelli) [reply modules/appeals_api/app/controllers/appeals_api/application_controller.rb]: whoops! yes we should (although I believe I'll do it in the contestable_issues_controller or service.rb)

Comment (edmangimelli) [reply modules/appeals_api/app/controllers/appeals_api/application_controller.rb]: or, it sounds like this is stripped out before it hits vets-api

File: lib/appeals/service.rb [hunk 2]
```diff
@@ -26,11 +28,28 @@ class Service < Common::Client::Base
     #
     def get_appeals(user, additional_headers = {})
       with_monitoring do
-        response = perform(:get, '/api/v2/appeals', {}, request_headers(user, additional_headers))
+        response = perform(
+          :get,
+          '/api/v2/appeals',
```
Comment (erluti) [inline lib/appeals/service.rb]: Can we make a constant for this too while we're changing it around?

**Pull request merged**